config.sh

```diff
@@ -30,7 +30,7 @@ AUTOMOUNT=true
 PROPFILE=true
 
 # Set to true if you need post-fs-data script
-POSTFSDATA=true
+POSTFSDATA=false
```

Comment:
You disabled a whole file you added a lot to.

Reply:
Oops